action.yml

```diff
@@ -6,11 +6,57 @@ branding:
   color: orange
 inputs:
   ocaml-version:
```

Comment:
Now, I think the input name `ocaml-compiler` is more accurate than `ocaml-version`.

Reply:
Done.